package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "start": "env NEXT_DEV=1 next",
     "build": "next build && next export",
-    "postbuild": "cp -Rv public/* public/.nojekyll out",
+    "postbuild": "cp -Rv public/* public/.nojekyll out && sw-precache --config=sw-precache-config.js --root=out/",
     "deploy": "gh-pages --src '{.nojekyll,**/*}' -d out -m \"Deploy $(git rev-parse HEAD) to GitHub pages [ci skip]\"",
     "lint": "eslint *.js components pages",
```

Comment:
Please adjust the linting script to check for files in the `static` and `public` folders as well.